content/powershell/concepts/conditionals/conditionals.md

```diff
@@ -1,13 +1,113 @@
 ---
 Title: 'Conditionals'
-Description: ''
+Description: 'The if, else, elseif, and switch conditionals in PowerShell are used for control flow. They allow us to choose which sections of code are executed based on conditions.'
 Subjects:
+  - 'Code Foundations'
   - 'Computer Science'
+  - 'Bash/Shell'
 Tags:
   - 'Conditionals'
-  - 'If'
+  - 'Control Flow'
   - 'Else'
+  - 'If'
+  - 'Switch'
 CatalogContent:
   - 'learn-powershell'
   - 'paths/computer-science'
 ---
+
+**Conditionals** enable execution of code based on provided conditions. They take in an expression and check if it is `True` or `False`. If the expression is `True`, one set of statements is executed. Otherwise, another set of instructions is performed. This control of flow makes programs more robust by enabling them to address multiple scenarios.
```

Comment:
Make sure True and False are capitalized (or not) across all docs. I feel like I came across some lower case ones.

Reply:
Thank you. I found some in the Operators doc.